src/schema/ecommerce/types/offer.ts

```diff
@@ -48,6 +49,10 @@ export const OfferType = new GraphQLObjectType({
         { rootValue: { userByIDLoader, partnerLoader } }
       ) => resolveOrderParty(from, userByIDLoader, partnerLoader),
     },
+    fromParticipant: {
+      type: OrderParticipantEnum,
+      description: "Offer is from which order participant",
```

Comment:
This wording is a little confusing -- what about something like "the order participant who created the offer"?

Reply:
makes total sesne